sdk/cosmosdb/cosmos/test/public/integration/aggregates/groupBy.spec.ts

```diff
@@ -640,12 +640,47 @@ describe("Cross partition GROUP BY", () => {
     );
   });
 
-  it("with multiple aggregates", async () => {
+  it.skip("with MakeList", async () => {
     const queryIterator = container.items.query(
-      "SELECT c.name, Count(1) AS count, Min(c.age) AS min_age, Max(c.age) AS max_age FROM c GROUP BY c.name",
+      "SELECT c.name, MakeList(c.age) AS ages FROM c GROUP BY c.name",
       options,
     );
     const result = await queryIterator.fetchAll();
+    for (const item of result.resources) {
+      item.ages = item.ages.sort();
+    }
+    snapshot(
+      result.resources.sort((a, b) => {
+        return a.name.localeCompare(b.name);
+      }),
+    );
+  });
+
+  it.skip("with MakeSet", async () => {
```

Comment:
Is there a reason these are added as `skipped`? Just curious

Reply:
this is waiting upon latest public emulator release.